marge/git.py

```diff
@@ -19,14 +19,14 @@
 
 
 def _filter_branch_script(trailer_name, trailer_values):
-    filter_script = 'TRAILERS={trailers} python3 {script}'.format(
-        trailers=shlex.quote(
-            '\n'.join(
-                '{}: {}'.format(trailer_name, trailer_value)
-                for trailer_value in trailer_values or [''])
-        ),
-        script=trailerfilter.__file__,
+    trailers = shlex.quote(
+        '\n'.join(
+            f'{trailer_name}: {trailer_value}'
+            for trailer_value in trailer_values or ['']
```

Comment:
(Not your fault, but this code really makes me feel pity for the Python parser... :grin: Trying to parse an incomplete loop comprehension without any semicolons or anything to help it keep track of where it ends. But I guess the closing parenthesis on line 26 is what makes it doable.)